packages/augur-sdk/src/api/MarketInvalidBids.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ParsedOrderEventLog,
+  OrderTypeHex,
+  SubscriptionEventName,
+} from '@augurproject/sdk-lite';
+import { Augur } from '../Augur';
+import * as _ from 'lodash';
+
+export class MarketInvalidBids {
+  private readonly augur: Augur;
+
+  constructor(augur: Augur) {
+    this.augur = augur;
+
+    this.augur.events.on(
+      SubscriptionEventName.BulkOrderEvent,
+      (orderEvents) => this.getMarketInvalidBids(orderEvents)
+    );
+  }
+
+  getMarketInvalidBids(orders: { logs: ParsedOrderEventLog[] }) {
+    const onlyInvalidBids = orders.logs.filter(
+      (o) => String(o.orderType) === OrderTypeHex.Bid && o.outcome === `0x00`
+    );
+    const markets = _.map(onlyInvalidBids, 'market');
+    const marketIds = Array.from(new Set(markets));
+    Promise.all(
```

Comment:
I think promise.all actually had a limit to the number of promises it handles..

@justinbarry was this some platform specific thing?